Description: Fix manpages.
 In particular, fix spelling-error-in-manpage, hyphen-used-as-minus-sign,
 and manpage-has-errors-from-man Lintian warnings.
Author: Doug Torrance
Last-Update: 2014-11-05

--- a/wmbiff/wmbiffrc.5.in
+++ b/wmbiff/wmbiffrc.5.in
@@ -24,7 +24,7 @@
 .SH OPTIONS
 Each option takes the form
 .IR option[.mbox] " = " value .
-Comments must be preceeded by pound signs (#).
+Comments must be preceded by pound signs (#).
 
 The supported configuration options are:
 
@@ -49,7 +49,7 @@
 is: \fINORMAL\fP.
 
 gnutls predefined values:
-.SP
+.sp
 .RS 8
 \fIPERFORMANCE\fP (gnutls >= 2.2.0)
 .RE
@@ -97,7 +97,7 @@
 pixmap file.
 .TP
 \fBglobalnotify\fP
-Command to be executed when new mail is recieved in any mailbox. Set
+Command to be executed when new mail is received in any mailbox. Set
 notify.n to override this option for mailbox n.
 .TP
 \fBlabel.n\fP
@@ -237,7 +237,7 @@
 checkings, default is the global interval.
 .TP
 \fBfetchinterval.n\fP
-Interval between mail auto-fetching. Values accept 0 to disable, -1 for
+Interval between mail auto-fetching. Values accept 0 to disable, \-1 for
 autofetching on new mail arrival, and positive values for a given interval
 in seconds.
 .TP
@@ -299,12 +299,12 @@
 failures.  Some servers claim to support cram-md5 but fail:
 telling wmbiff not to try can help.
 
-For other problems, run wmbiff with the -debug option.  See
+For other problems, run wmbiff with the \-debug option.  See
 wmbiff(1) for details.
 
 While editing .wmbiffrc, you may find it useful to restart
 wmbiff using either control-shift mouse button 1, or killall
--USR1 wmbiff.
+\-USR1 wmbiff.
 
 
 .SH FILES
--- a/wmbiff/wmbiff.1
+++ b/wmbiff/wmbiff.1
@@ -13,7 +13,7 @@
 
 .SH SYNOPSIS
 .B wmbiff
-[-display <display name>] [-geometry +XPOS+YPOS] [-c <filename>] [-h] [-v] [-debug] [-fg <foreground-color>] [-bg <background-color>] [-hi <highlight-color>] [-font <X11 font>|default] [-o] [+w]
+[\-display <display name>] [\-geometry +XPOS+YPOS] [\-c <filename>] [\-h] [\-v] [\-debug] [\-fg <foreground-color>] [\-bg <background-color>] [\-hi <highlight-color>] [\-font <X11 font>|default] [\-o] [+w]
 .br
 
 .SH DESCRIPTION
@@ -60,15 +60,15 @@
 Print verbose log of progress.
 .TP
 .B \-fg <color>
-Use specified X11 color for foreground. Implies -font default,
+Use specified X11 color for foreground. Implies \-font default,
 unless overridden.
 .TP
 .B \-bg <color>
-Use specified X11 color for background. Implies -font default,
+Use specified X11 color for background. Implies \-font default,
 unless overridden.
 .TP
 .B \-hi <color>
-Use specified X11 color for new mail counters. Implies -font
+Use specified X11 color for new mail counters. Implies \-font
 default, unless overridden.
 .TP
 .B \-font <font>
@@ -97,7 +97,7 @@
 Do not use the "withdrawn" state: the wmbiff window will not
 be captured as an icon and placed in the dock, but will
 instead have its own window.  This option starts with a '+'
-for consistency with other software programs: many use '-w'
+for consistency with other software programs: many use '\-w'
 to mean the opposite.
 
 .SH BUGS
@@ -105,7 +105,7 @@
 Mailing List <wmbiff-devel@lists.sourceforge.net>.   Consider
 attaching a transcript of your session, generated using:
 .RS
-wmbiff -debug | tee wmbiff-log
+wmbiff \-debug | tee wmbiff-log
 .RE
 Be sure to remove any instances of your password.
 
